FrontEnd/src/components/AboutPage/MeetTheTeam/MeetTheTeam.tsx

```diff
@@ -1,7 +1,126 @@
 import React, { FunctionComponent } from 'react';
 import { Col, Container, Row } from 'react-bootstrap';
 import styles from './MeetTheTeam.module.scss';
+import MemberInfoCard, {
+  MemberInfoCardProps,
+} from './MemberInfoCard/MemberInfoCard';
 
-const MeetTheTeam: FunctionComponent = () => <Container></Container>;
+const Cards: MemberInfoCardProps[] = [
+  {
+    name: 'Will Hsu',
+    imgSrc:
+      'https://drive.google.com/uc?export=view&id=1Gyusdebr_JVrgGYllwV5ODkNIAZot6ca',
+    title: 'Developer',
+
```

Comment:
Is your prettier working? i don't think there should be a new line here if it is. If your prettier isn't working, we need to get it working and run it on each file you altered.